dotnet/src/Functions/Functions.OpenApi/Model/RestApiParameter.cs

```diff
@@ -16,9 +17,19 @@ public sealed class RestApiParameter
     public string Name { get; }
 
     /// <summary>
-    /// The property alternative name. It can be used as an alternative name in contexts where the original name can't be used.
+    /// The parameter argument name.
+    /// If provided, the argument name will be used to search for the parameter value in function arguments.
+    /// If no value is found using the argument name, the original name - <see cref="RestApiParameter.Name"/> will be used for the search instead.
     /// </summary>
-    public string? AlternativeName { get; internal set; }
+    public string? ArgumentName
```

Comment:
Not sure if it's a scope of this PR, but would be great to have an example with this functionality in action.

Reply:
There's no place in SK where it's possible to access the model yet. This change, with sample, will come in the next PR.